.eslintrc.json

```
@@ -21,6 +21,7 @@
         "padded-blocks": "off",
         "no-param-reassign":"off",
         "max-len":"off",
-        "no-console":"off"
+        "no-console":"off",
+        "eslint-disable import/prefer-default-export":"off"
```

```suggestion
        "no-console": "off",
        "eslint-disable import/prefer-default-export": "off"
```
Do this for line 22 and line 23 as well...makes it more readable